src/main/java/console_app/PomodoroController.java

```diff
@@ -0,0 +1,42 @@
+package main.java.console_app;
+
+import main.java.services.pomodoro_running.CancelTimerInput;
+import main.java.data_gateway.PomodoroTimerManager;
+import main.java.services.pomodoro_running.PomodoroObserver;
+import main.java.services.pomodoro_running.PomodoroRunner;
+
+public class PomodoroController {
+    private PomodoroRunner pomodoroRunner;
+    private final PomodoroTimerManager pomodoroTimerManager = new PomodoroTimerManager();
+    private boolean switched = false;
```

Comment:
The meaning of `switched` is not clear.
Can you change the attribute name or add documentation for `switched`?

Reply:
^^